M2/configure.ac

```diff
@@ -1139,8 +1139,8 @@ AC_LANG(C++)
 AC_CHECK_HEADER(frobby.h,LIBS="-lfrobby $LIBS",BUILD_frobby=yes)
 if test $BUILD_frobby = yes
 then BUILTLIBS="-lfrobby $BUILTLIBS"
-    dnl change this to [1] when frobby is built from a submodule
-    AC_DEFINE([HAVE_FROBBY_VERSION], [0], [whether frobby has frobby_version])
+    dnl uncomment this to [1] when frobby is built from a submodule (?)
+    dnl AC_DEFINE([HAVE_FROBBY_VERSION], [0], [whether frobby has frobby_version])
```

Comment:
Did you test this with both the old version and the new version of Frobby?

Reply:
There's nothing to test -- the line is active only when we are building frobby, and the version to build is hard-coded in frobby/Makefile.in.

Also, `#define HAVE_FROBBY_VERSION 0` doesn't have the desired effect on your line `#ifdef HAVE_FROBBY_VERSION` in `version.dd`.